### DISCUSSION TITLE
The PrismNet doen't have user mmccants anymore

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3819] [General]
* Stellarium version: <Stellarium 24.2+ (v24.2.201-e0f00c3 [master]>

log.txt: [Satellites] FAILED to download "https://www.prismnet.com/~mmccants/tles/classfd.zip" Error: "Error transferring https://www.prismnet.com/~mmccants/tles/classfd.zip - server replied: Not Found"

This error or warning occurs because PrismNet doesn't have user mmccants anymore or at this moment.

**Comment by alex-w** [comment]
Please reset the default list of sources in Satellites plugin

**Reply by ilkant** [reply]
Thanks.
